aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/SdkSettings.kt

```diff
@@ -47,11 +45,10 @@ class SdkSettings private constructor(private val awsSdk: ObjectNode?) {
 
     /** Whether to generate a README */
     val generateReadme: Boolean
-        get() =
-            awsSdk?.getBooleanMember("generateReadme")?.orNull()?.value ?: false
+        get() = awsSdkBuild && !(awsSdk?.getBooleanMember("suppressReadme")?.orNull()?.value ?: false)
 
     val requireEndpointResolver: Boolean
-        get() = awsSdk?.getBooleanMember("requireEndpointResolver")?.orNull()?.value ?: true
```

Comment:
We should still look up these old values and log warnings that they're no longer read since they're used by a lot of people.